site/documentation/content/api/Metrics.md

```diff
@@ -75,6 +75,7 @@ Metrics provided by the protocol adapters are:
 | *hono.connections.authenticated*   | Gauge               | *host*, *component-type*, *component-name*, *tenant*                                         | Current number of connected, authenticated devices. <br/> **NB** This metric is only supported by protocol adapters that maintain *connection state* with authenticated devices. In particular, the HTTP adapter does not support this metric. |
 | *hono.connections.unauthenticated* | Gauge               | *host*, *component-type*, *component-name*                                                   | Current number of connected, unauthenticated devices. <br/> **NB** This metric is only supported by protocol adapters that maintain *connection state* with authenticated devices. In particular, the HTTP adapter does not support this metric. |
 | *hono.connections.authenticated.duration* | Timer        | *host*, *component-type*, *component-name*, *tenant*                                         | The overall amount of time that authenticated devices have been connected to protocol adapters. <br/> **NB** This metric is only supported by protocol adapters that maintain *connection state* with authenticated devices. In particular, the HTTP adapter does not support this metric. |
+| *hono.connections.attempts.total*  | Counter             | *host*, *component-type*, *component-name*, *outcome*                                        | The number of connection attempts to protocol adapters. The outcome can be observed by looking the tag *outcome* which contains the result and the reason. At the moment there's only one possible value: `adapter-connection-limit-exceeded`<br/> **NB** This metric is only supported by protocol adapters that maintain *connection state* with authenticated devices. In particular, the HTTP adapter does not support this metric. |
```

Comment:
according to the constant definition in `MicrometerBasedMetrics`, the name is `hono.connections.attempts` only (which I find preferable).

Reply:
Well, actually Prometheus adds the "_total" suffix (see https://prometheus.io/docs/practices/naming/#metric-names).

Reply:
That might be true, but this spec is about the metrics that we report using the Micrometer framework in the Hono components. Prometheus is one of multiple options for a metrics backend where the meter readings are being collected. We have little (that is: no) influence on what these systems do with the data. However, this specification should define the names that we use in our components only. Does that make sense?